test/test.py

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
Funny, don't know how it ended up here. I think I mixed up two branches earlier...